package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/js-client-sdk-common",
-  "version": "4.3.0",
+  "version": "4.5.0",
```

Comment:
Since this is not adding new (from the users perspective) functionality but instead just adjusting something existing (similar to a bugfix) let's just bump the patch number by one: `4.3.1`. (if this is out of date, rebase on latest main)